<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>2200</width>
    <height>1300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>img/plhlder.png</normaloff>img/plhlder.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(77, 77, 127);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>1200</width>
     <height>800</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QLabel" name="stream_section">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>0</y>
      <width>971</width>
      <height>671</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
QLabel#stream_section {
	image: url(:/images/img/plhlder.png);
	border-radius: 15px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QFrame" name="controls_section">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>0</y>
      <width>231</width>
      <height>801</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame#controls_section {
background-color: qlineargradient(spread:pad,y1:0, x1:0, y1:0, x1:1, stop:0 rgb(255, 255, 255, 1), stop:1 rgba(40, 183, 209));
	border-top-right-radius: 15px;
	border-bottom-left-radius: 15px;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="circularProgressBar_Main_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>410</y>
       <width>240</width>
       <height>240</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: none;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QFrame" name="circularProgressRAM">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>220</width>
        <height>220</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame{
	border-radius: 110px;	
	background-color: qconicalgradient(cx:0.5, cy:0.5, angle:90, stop:0.750 rgba(255, 0, 127, 255), stop:0.745 rgba(255, 255, 255, 0));
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QFrame" name="circularBg_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>220</width>
        <height>220</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame{
	border-radius: 110px;	
	background-color: rgba(85, 85, 127, 100);
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QFrame" name="circularContainer_3">
       <property name="geometry">
        <rect>
         <x>14</x>
         <y>16</y>
         <width>190</width>
         <height>190</height>
        </rect>
       </property>
       <property name="baseSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QFrame{
	border-radius: 95px;	
	background-color: rgb(58, 58, 102);
}</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QWidget" name="layoutWidget_3">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>30</y>
          <width>191</width>
          <height>137</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="infoLayout_3">
         <item row="0" column="0">
          <widget class="QLabel" name="labelAplicationName_3">
           <property name="font">
            <font>
             <family>Helvetica Neue</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: #FFFFFF; background-color: none;</string>
           </property>
           <property name="text">
            <string>&lt;strong&gt;RAM&lt;/strong&gt; USAGE</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="labelCredits_3">
           <property name="font">
            <font>
             <family>Helvetica Neue</family>
             <pointsize>8</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(148, 148, 216); background-color: none;</string>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="labelPercentageRAM">
           <property name="font">
            <font>
             <family>Helvetica Neue</family>
             <pointsize>30</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 44, 174); padding: 0px; background-color: none;</string>
           </property>
           <property name="text">
            <string>&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:50pt;&quot;&gt;25&lt;/span&gt;&lt;span style=&quot; font-size:40pt; vertical-align:super;&quot;&gt;%&lt;/span&gt;&lt;/p&gt;</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="indent">
            <number>-1</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QFrame" name="circularProgressBar_Main">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>130</y>
       <width>240</width>
       <height>240</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: none;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QFrame" name="circularProgressCPU">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>220</width>
        <height>220</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame{
	border-radius: 110px;	
	background-color: qconicalgradient(cx:0.5, cy:0.5, angle:90, stop:0.400 rgba(85, 170, 255, 255), stop:0.395 rgba(255, 255, 255, 0));
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QFrame" name="circularBg">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>220</width>
        <height>220</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame{
	border-radius: 110px;	
	background-color: rgba(85, 85, 127, 100);
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QFrame" name="circularContainer">
      <property name="geometry">
       <rect>
        <x>25</x>
        <y>25</y>
        <width>190</width>
        <height>190</height>
       </rect>
      </property>
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame{
	border-radius: 95px;	
	background-color: rgb(58, 58, 102);
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>40</y>
         <width>191</width>
         <height>137</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="infoLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="labelAplicationName">
          <property name="font">
           <font>
            <family>Helvetica Neue</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: #FFFFFF; background-color: none;</string>
          </property>
          <property name="text">
           <string>&lt;strong&gt;CPU&lt;/strong&gt; USAGE</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="labelPercentageCPU">
          <property name="font">
           <font>
            <family>Helvetica Neue</family>
            <pointsize>30</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(115, 185, 255); padding: 0px; background-color: none;</string>
          </property>
          <property name="text">
           <string>&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:50pt;&quot;&gt;60&lt;/span&gt;&lt;span style=&quot; font-size:40pt; vertical-align:super;&quot;&gt;%&lt;/span&gt;&lt;/p&gt;</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="indent">
           <number>-1</number>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="labelCredits">
          <property name="font">
           <font>
            <family>Helvetica Neue</family>
            <pointsize>8</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(148, 148, 216); background-color: none;</string>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <zorder>circularBg</zorder>
     <zorder>circularProgressCPU</zorder>
     <zorder>circularContainer</zorder>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>760</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: none;
color: rgb(58, 58, 102);
</string>
     </property>
     <property name="text">
      <string>Processor</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>740</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: none;
color: rgb(58, 58, 102);

</string>
     </property>
     <property name="text">
      <string>System</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="controls_section_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>680</y>
      <width>991</width>
      <height>121</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame#controls_section_2 {
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgb(255, 255, 255, 1), stop:1 rgba(40, 183, 209));
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="midLineWidth">
     <number>0</number>
    </property>
    <widget class="QPushButton" name="start_btn">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>10</y>
       <width>101</width>
       <height>101</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#start_btn {
	color: rgb(187, 170, 244);
	font-weight: bold;
	border-radius: 25px;
}

QPushButton#start_btn:hover{
 	padding: -5px;
	border-radius: 25px;
	background-color: rgb(52, 41, 86);
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="imges_rc.qrc">
       <normaloff>:/images/img/start.png</normaloff>:/images/img/start.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="stop_btn">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>10</y>
       <width>101</width>
       <height>101</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#stop_btn {
	color: rgb(187, 170, 244);
	font-weight: bold;
	border-radius: 25px;
}

QPushButton#stop_btn:hover{
 	padding: -5px;
	border-radius: 25px;
	background-color: rgb(52, 41, 86);
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="imges_rc.qrc">
       <normaloff>:/images/img/stop-1.png</normaloff>:/images/img/stop-1.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QProgressBar" name="progressBar_2">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>10</y>
      <width>140</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>140</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>140</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QProgressBar {
	background-color: rgb(58, 58, 102, 120);
	border-style: none;
	border-radius: 8px;
	text-align: center;
	font-weight: bold;
	color: #fff;

}
QProgressBar::chunk{
	background-color: rgb(115, 210, 22);
    border-radius: 4px;
    width: 25px;
    margin: 2px;
}</string>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>50</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignHCenter</set>
    </property>
    <property name="textVisible">
     <bool>true</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="textDirection">
     <enum>QProgressBar::TopToBottom</enum>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>9</y>
      <width>291</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;
background: none;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 34px;
font-weight: bold;
color: rgb(39, 192, 221, 70);
background: none;
</string>
     </property>
     <property name="text">
      <string>Live</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>73</x>
       <y>9</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 40px;
font-weight: bold;
color: rgb(39, 192, 221, 40);
background: none;
</string>
     </property>
     <property name="text">
      <string>Streamer</string>
     </property>
    </widget>
   </widget>
   <zorder>stream_section</zorder>
   <zorder>controls_section_2</zorder>
   <zorder>controls_section</zorder>
   <zorder>progressBar_2</zorder>
   <zorder>frame</zorder>
  </widget>
 </widget>
 <resources>
  <include location="imges.qrc"/>
  <include location="imges_rc.qrc"/>
 </resources>
 <connections/>
</ui>
